dfs/nfs/server.go

```diff
@@ -398,6 +398,40 @@ func (c *Server) cleanupBindMounts() {
 			}
 		}
 	}
+	// CC-1816: clean up remnants of old /exports/serviced_var_volumes directory
+	removeDeprecated("/exports/serviced_var_volumes")
+}
+
+// removeDeprecated will clean up any old exports path
+func removeDeprecated(dirpath string) {
+	if dirContents, err := ioutil.ReadDir(dirpath); !os.IsNotExist(err) {
+		if err != nil {
+			glog.Warningf("Could not look up deprecated exports path %s: %s", dirpath, err)
```

Comment:
I set everything to warning because the caller doesn't set a return value, I was maintaining consistency.